addOns/spiderAjax/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 - Now using 2.10 logging infrastructure (Log4j 2.x).
 - Maintenance changes.
+- Remove automation parameters `failIfFoundUrlsLessThan` and `warnIfFoundUrlsLessThan`
+  in favour of the `spiderAjax.urls.added` statistic test.
```

Comment:
The add-on will also need to depend on the new version on the automation add-on.